wp-content/plugins/tapestry/tapestry.php

```diff
@@ -87,4 +84,16 @@ function add_my_post_types_to_query($query) {
     }
     return $query;
 }
-add_action('pre_get_posts', 'add_my_post_types_to_query');
\ No newline at end of file
+add_action('pre_get_posts', 'add_my_post_types_to_query');
+
+/**
+ * Filter the template for Tapestry post
+ */
+function load_tapestry_template($singleTemplate) {
+    global $post;
```

Comment:
I followed the example provided by Wordpress. Do you think it's good to follow them?
https://codex.wordpress.org/Plugin_API/Filter_Reference/single_template

Reply:
Yeah this looks like the correct implementation.